### ISSUE TITLE
Doesn't work with next 11

### ISSUE THREAD

**Issue by johnschult** [devrnt/react-use-wizard#52]
I ran the `next` example using this `package.json` and you cannot advance to the next step.

```json
{
  "name": "nextjs",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {
    "next": "^11",
    "react": "^17",
    "react-dom": "^17",
    "react-use-wizard": "^1"
  }
}
```

![Screen Shot 2021-07-15 at 8 15 36 AM](https://user-images.githubusercontent.com/9453/125786481-8c322aaa-3f44-43a9-ba12-0c5ba8f1a53a.png)

After clicking Next...

![Screen Shot 2021-07-15 at 8 15 49 AM](https://user-images.githubusercontent.com/9453/125786527-d4e60423-faff-48ff-9e4a-08c4c70f8c26.png)

No console messages or errors.

**Comment by johnschult** [comment]
Solved by changing the calls to `nextStep` and `previousStep`...

```javascript
<div>
  <button
    onClick={() => previousStep()}
    disabled={isLoading || isFirstStep}
  >
    Previous
  </button>
  <button onClick={() => nextStep()} disabled={isLoading || isLastStep}>
    Next
  </button>
</div>
```

**Comment by devrnt** [comment]
Good catch, will fix the examples. It's because the event is being passed implicitly.

**Comment by devrnt** [comment]
This was working in previous versions since you couldn't pass an index to `previousStep` and `nextStep`

**Comment by devrnt** [comment]
Closed in #53
